db/middleware.go

```diff
@@ -127,8 +127,8 @@ func transformQuery(queryParam url.Values) (url.Values, error) {
 }
 
 // postgrestValues returns ["a", "b", "c"] as `"a","b","c"`
-func postgrestValues(val []string) string {
```

Comment:
@moshloop  Are these changes okay ? there was a change in duty here: https://github.com/flanksource/duty/commit/f38535fef4cb60f48c0236a430d2978a85132c64